src/backend/api/reservations.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const { max } = require("../database");
+const router = express.Router();
+const knex = require("../database");
+//Reservation
+//	Returns all reservations
+router.get("/", async (request, response) => {
+    let myTable = await knex ("Reservation");
```

Comment:
I'm not an expert on knex but shouldn't this code be like:
`const reservations = await knex('Reservation').select('*');`